src/main/scala/net/ceedubs/ficus/readers/AnyValReaders.scala

```diff
@@ -1,6 +1,6 @@
 package net.ceedubs.ficus.readers
 
-import com.typesafe.config.Config
+import com.typesafe.config.{ConfigException, Config}
```

Comment:
Not a big deal, but this import can be reverted since the Big* readers are no longer in this file.